app/helpers/course/controller_helper.rb

```diff
@@ -0,0 +1,30 @@
+module Course::ControllerHelper
+  # Formats the given +CourseUser+ as a user-visible string.
+  #
+  # @param [CourseUser] user The User to display.
+  # @return [String] The user-visible string to represent the User, suitable for rendering as
+  #   output.
+  def display_course_user(user)
+    user.name
```

Comment:
@lowjoel Shall we display `course_user.user.name` here ?  Thus all the names will be consistent.  Or just leave it here and change in future when needed ?